fuseimg/data/ops/color.py

```diff
@@ -63,8 +62,7 @@ def __init__(self, **kwargs):
 
     def __call__(self, sample_dict: NDict, key: str):
         img = sample_dict[key]
-        img = skimage.img_as_float(img)
-        sample_dict[key] = img
+        sample_dict[key] = img / img.max()
```

Comment:
Cool I remember we talked about changing that :) 
Shouldn't we also subtract the min value? When the minimum value isn't zero the range won't be exactly [0, 1]

Reply:
Good catch! - I copied it from ukbb branch without checking.